lightning/src/ln/channelmonitor.rs

```diff
@@ -2394,6 +2394,10 @@ impl<ChanSigner: ChannelKeys> ChannelMonitor<ChanSigner> {
 		}
 	}
 
+	/// Called by ChannelMonitor::block_connected, which implements ChainListener::block_connected.
```

Comment:
"Should be called by ManyChannelMonitor implementation"

Reply:
Sadly, it can't be - its not pub (I have a patch for this, but too many in-flight PRs right now). Its only called by SimpleManyChannelMonitor  (which is a typo here, now fixed).